### DISCUSSION TITLE
RPC question

### DISCUSSION THREAD

**Discussion by zhang-zimin** [iTwin/itwinjs-core#7449] [Q&A] [answered]
Hi, I want to use RPC for interaction with the client. Is there any sample I can refer to?

**Comment by paulius-valiunas** [comment, marked as answer]
Sorry the documentation is a bit messy. Everything should be on [this page](https://www.itwinjs.org/learning/rpcinterface/#server-side-configuration). If you just want to use our cloud backend, skip straight to client-side configuration. If you want to serve the same interfaces from your own backend, start with server-side configuration. If you want to add your own custom functionality to your backend, you can start from the top of the page, but bear in mind we are beginning to phase out RPC and I wouldn't recommend writing any new RPC interfaces. In the future we will be replacing all backend-frontend communication with HTTPS and IPC APIs. For any new functionality, I would recommend implementing it using either IPC (works on Electron and mobile) or using a 3rd party solution to build an HTTP API. We will be releasing more specific guidelines and documentation early next year.
